### PR TITLE
Results design polish

### PR DESCRIPTION
**Pull request by Gerg**
Did a design pass on the results with the goal to make them more compact and a tad more pretty.

1) Added new svg friends icon to
- Results courses
- Results sections
- Friends list
- Notifications tray

2) Reduced margins and padding for section and course results

3) Hide results section friends list when no friends in section

4) Replaced JQuery slideToggle with custom animation because it was hitching (apparently a known issue if your elements have negative margins or something that hasn't been fixed in 3 years)

### CODE REVIEW

File: public/scripts/backbone-schedule/models/section.coffee [hunk 1]
```diff
@@ -17,8 +17,8 @@ define(['jQuery'
     __rnd: (seed = Date.now()) ->
       ((seed*9301+49297) % 263212) / (263212.0)
 
-    description: ->
-      @.get('description') || @.get('courseDescription') || @.get('course')?.get('description') || "No description available"
```
Comment (jrundquist) [inline public/scripts/backbone-schedule/models/section.coffee:21]: Is there a reason `@.get('course')?.get('description')` was removed?

Comment (Gerg) [inline public/scripts/backbone-schedule/models/section.coffee:21]: .get('course') just gets the id of the course, so it doesn't work.  @.get('courseDescription') is populated with that information anyway, so it was broken and seemed unnecessary.

Comment (jrundquist) [inline public/scripts/backbone-schedule/models/section.coffee:21]: Yeah, The original intent was that if the course was populated with an object to return its description and eventually phase out 'courseDescription', but for now this is good.

**Comment by jrundquist** [comment]
Nice new friends icon.

I would suggest aligning the icon ( in the scope of the section itself) to the right-side. That seems to be where i look for it every time. ( probably b/c thats where it is on the course block )

Other than that it looks good to go.

**Comment by jrundquist** [comment]
**Results list look**
Screen Shot 2012-12-17 at 12 13 25 PM

**Pull request merged**